public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -96,6 +96,7 @@ export class DashboardModel {
   panels: PanelModel[];
   panelInEdit?: PanelModel;
   panelInView?: PanelModel;
+  fiscalYearStartMonth?: number;
```

Comment:
@hugohaggmark mind we are adding this new property to the DashboardModel.

Reply:
@dprokop thanks for heads up, does `0` mean January i.e. we're following default JavaScript month numbering confusion?

Reply:
Yes 0 is January. I was playing with the idea of having a month enum so we don't end up with negative or > 11 values. It would still be a 0 for Jan in the dashboard json though. Another option would be to hide it in month class with a string represenation for the json and then cast it when we need it for the moment.js integration. Feels a bit over engineered though

Reply:
Thanks for explaining that, maybe it's worth adding a brief comment about it?